tests/Signals_perftest.m

```diff
@@ -12,11 +12,9 @@
 % `A` & `B` properties are treated as "input layer" origin signals from 
 % which all other signals are created.
 %   
-% Example - display the mean measured time for running each test:
+% Example:
```

Comment:
I guess you removed the documentation for this example because `sampleSummary` is pretty straightforward, but in general in docstrings it's nice to detail what the example does